script/github-actions/remove-global-assets.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
```

Comment:
I was thinking about making this a node script, but it was just easier and quicker in bash. And this avoids needing to install dependencies in the `Archive` job.